packages/@dcl/inspector/src/components/ProjectAssetExplorer/ProjectAssetExplorer.css

```diff
@@ -1,53 +1,11 @@
-.asset-explorer-container {
-  display: flex;
-  flex-wrap: wrap;
-  height: 100%;
-  overflow: auto;
+.editor .testing .item {
+  padding: 16px;
 }
 
-.asset-explorer-left {
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-  background-color: #d34d4d;
+.editor .Tree {
+  border: none;
 }
 
-.asset-explorer-top {
-  display: flex;
-  flex-direction: column;
-  padding: 20px;
-  background-color: #11a661;
-  flex-grow: 1;
-}
-
-.asset-explorer-bottom {
-  display: flex;
-  padding: 10px;
-  background-color: #541a43;
-}
-
-.asset-explorer-scrollable {
-  overflow-y: auto;
-}
-
-.asset-explorer-item {
-  display: flex;
-  align-items: center;
-  padding: 10px;
-  cursor: pointer;
-}
-
-.asset-explorer-item:hover {
-  background-color: #bfcb1b;
-}
-
-.asset-explorer-item i {
-  margin-right: 10px;
-}
-
-.asset-explorer-right {
-  flex: 1;
-  background-color: #088270;
-  padding: 20px;
-  overflow: scroll;
+.editor .Tree span {
```

Comment:
These styles are dependant on parent classes, that could lead to errors if the on this component if the parent component changes. We usually let all the rules in a component css to be scoped to the component's outter class, which should match the name of the component, ie: `.ProjectAssetExplorer`